src/main/java/org/apache/maven/buildcache/xml/CacheConfigImpl.java

```diff
@@ -503,6 +503,11 @@ public String getAlwaysRunPlugins() {
         return getProperty(ALWAYS_RUN_PLUGINS, null);
     }
 
+    @Override
+    public boolean isMandatoryClean() {
+        return getConfiguration().isMandatoryClean();
```

Comment:
It might be handy to add a command line property as well

Reply:
Done :)